scale/job/execution/configuration/configurators.py

```diff
@@ -386,7 +386,7 @@ def _configure_regular_job(config, job_exe, job_type):
         """
 
         config.create_tasks(['pull', 'pre', 'main', 'post'])
-        config.add_to_task('pull', args=create_pull_command(job_type.get_tagged_docker_image()))
+        config.add_to_task('pull', args=create_pull_command(job_exe.job.job_type_rev.docker_image))
```

Comment:
This will result in 2 database queries for every job_exe scheduled. Recommend adding `docker_image` field to the queue model so it is available for scheduling without additional queries.